example/test_problem/Hydro/CCSN/Input__Parameter.Lightbulb

```diff
@@ -188,6 +188,7 @@ GREP_MAXRADIUS               -1.0            # maximum radius in the radial prof
                                              # (<=0=distance of the farthest vertex to the center)
 GREP_MINBINSIZE              -1.0            # minimum bin size                                             [-1.0]
                                              # (<=0=use amr->dh[MAX_LEVEL])
+GREP_OPT_FIXUP                0              # fix up the profiles on the father level at previous time     [0]
 
```

Comment:
This is minor but it is better to use "parent" and "child" instead of "father" and "son".

Reply:
I will update it later.